docs/troubleshooting.md

```diff
@@ -12,12 +12,14 @@ This can have several reasons:
   - Restarting promtail will not necessarily resend log messages that have been read. To force sending all messages again, delete the positions file (default location `/tmp/positions.yaml`) or make sure new log messages are written after both promtail and Loki have started.
 - Promtail is ignoring targets because of a configuration rule
   - Detect this by turning on debug logging and then look for `dropping target, no labels` or `ignoring target` messages.
+- Promtail cannot find the location of your log files. Check that the scrape_configs contains valid path setting for finding the logs in your worker nodes.
+- Your pods are running but not with the labels Promtail is expecting. Check the Promtail scape_configs.
```

Comment:
those part is a little duplicate with https://github.com/grafana/loki/blob/master/docs/troubleshooting.md#connecting-to-a-promtail-pod-to-troubleshoot

Reply:
The first (cannot find the location of your log files) reason was added based on the discussion on the slack where one user had this problem.

I added the second bullet based how I was personally hit with the fact that not all my streams were visible because I wasn't using compatible labels.

Reply:
The "Connecting to a promtail pod to troubleshoot" is a good chapter with further explanation how to debug the subject. I feel that mentioning these two cases specifically adds valuable information.